### PR TITLE
docs(contributing): create issue template for stories

### LINKED ISSUE #55: docs(contributing): create story issue template
Based on discussion #54 a more agile development is being taken for managing Megamind. A part of that will be using specific issues types, called stories, to track work. Reiterating from discussion #54, the story creation process will then flow as follows:

1. User creates generic issue (ideally following the conventional commit styling)
    a. Project members who are experienced in our agile flow may skip this and instead head straight to step 2
2. Project member reviews generic issue and if its determined to add value, then a story issue is created to replace it.
3. The new story will be assigned to the [Megamind Refinement](https://github.com/orgs/z5labs/projects/7) project.
    a. Then project members will "regularly" meet to refine the stories until they believe anyone can read the story and complete the work
4. Once the story is ready it will be transitioned to the main project board, [Megamind](https://github.com/orgs/z5labs/projects/3/views/1) and assigned to one of the milestones
5. At this point, the story will just remain in the project backlog until someone decides to pick it up in a sprint

### PR DESCRIPTION
**Pull request by Zaba505**
Closes #55

### CODE REVIEW

**Comment by nikitawootten** [comment]
For issue templates I'm a big fan of issue forms:  https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by Zaba505** [comment]
> For issue templates I'm a big fan of issue forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

@nikitawootten  Ooo I like that too. I see though it's only in beta for public projects. Should we wait till it stabilizes or do you think it's already stable enough?

**Comment by nikitawootten** [comment]
> > For issue templates I'm a big fan of issue forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
> 
> @nikitawootten  Ooo I like that too. I see though it's only in beta for public projects. Should we wait till it stabilizes or do you think it's already stable enough?

Oh it's pretty stable imo, my team had been using it for like 4 months without any breaking changes. I can PR this PR if you'd like

**Comment by Zaba505** [comment]
Nah I can refactor it. Should have it done soon

File: .github/ISSUE_TEMPLATE/story.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Story
+description: Represents stories
+title: "story(subject): short description"
+labels: ["story"]
+body:
+  - type: textarea
```
Comment (nikitawootten) [inline .github/ISSUE_TEMPLATE/story.yaml:20]:
```suggestion
  - type: markdown
```
Text areas are only suitable for short responses.

File: .github/ISSUE_TEMPLATE/story.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    id: description
+    attributes:
+      label: Description
+      description: Please provide an informal, general explanation of the work needed to be done, written from the perspective of the end user.
+    validations:
+      required: true
+
+  - type: textarea
```
Comment (nikitawootten) [inline .github/ISSUE_TEMPLATE/story.yaml:28]:
```suggestion
  - type: markdown
```
Markdown must be used for list content.

File: .github/ISSUE_TEMPLATE/story.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance Criteria
+      description: List the criteria for accepting this story as done
+    validations:
+      required: true
+
+  - type: textarea
```
Comment (nikitawootten) [inline .github/ISSUE_TEMPLATE/story.yaml:36]:
```suggestion
  - type: markdown
```
GH flavored markdown allows you to reference issues by #. Text areas are not suitable for this input type.

**Review by nikitawootten — CHANGES_REQUESTED** [review]
See suggestions below.

Edit: turns out I'm too tired to comprehend the GitHub form yaml, looks good.!

**Review by nikitawootten — APPROVED** [review]

**Comment by Zaba505** [comment]
Closed #55

**Pull request merged**